Frameworks/include/ShareSheetControl.h

```diff
@@ -0,0 +1,32 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+#pragma once
+
+#include <unknwn.h>
+#include "Social.Xaml.Export.h"
+
+struct IInspectable;
+
+class __declspec(novtable) __declspec(uuid("5CF22688-7892-4CE0-ACA4-3EA01A0EDDDB")) ShareSheetControl : public IUnknown {
```

Comment:
It's not a WinRT object. This interface is just used to bridge the gap between the C++/CX code in Social.Xaml.dll and the Objective-C++ code in Social.dll.

The only reason I made it a COM object at all is because I wanted its lifetime semantics to be easily understood.

---
In reply to: [107999932](https://github.com/Microsoft/WinObjC/pull/2314#discussion_r107999932) [](ancestors = 107999932,107999827)